src/styles.scss

```diff
@@ -2,19 +2,40 @@
 @use 'themes/common/typography';
 @use 'themes/common/theming';
 @use 'themes/default';
+@use 'themes/dark';
+@use 'app/modules/ontology-exploration/ontology-search/ontology-search.component.theme' as search;
 
 @include ng.mat-core(typography.$config);
 
-@include theming.ccf-theme(default.$theme);
+// .cdk-overlay-container {
+//   &.dark-theme {
+//     @include theming.ccf-theme(dark.$theme);
+//   }
+// }
 
-/**
- * Additional themes can be placed here. Example:
- *
- * .my-other-theme {
- *   @include theming.ccf-theme(other.$theme);
- * }
- *
- */
+@include theming.ccf-theme(dark.$theme);
```

Comment:
Don't include this twice! Instead set the default (dark) theme in the app module by providing a value for DEFAULT_THEME (from theming service)